bingo/symbolic_regression/symbolic_regressor.py

```diff
@@ -157,136 +223,63 @@ def fit(self, X, y, sample_weight=None):
             print("sample weight not None, TODO")
             raise NotImplementedError
 
-        n_cpus = os.environ.get('OMP_NUM_THREADS', 1)
+        n_cpus = int(os.environ.get("OMP_NUM_THREADS", "0"))
 
-        print(f"using {n_cpus} processes")
         self.archipelago = self._get_archipelago(X, y, n_cpus)
-        print(f"archipelago: {type(self.archipelago)}")
 
         max_eval_scaling = 1
         if isinstance(self.archipelago, FitnessPredictorIsland):
-            print("n_points per predictor:", self.archipelago._predictor_size)
             if self.scale_max_evals:
-                max_eval_scaling = len(X) / self.archipelago._predictor_size / 1.1
-        # print("max evals:", self.max_evals * max_eval_scaling)
+                max_eval_scaling = \
+                    len(X) / self.archipelago._predictor_size / 1.1
```

Comment:
Likewise I don't fully understand this scaling.

Reply:
since fitness predictors only use "partial" evals we're scaling up the evals to allow for the equivalent of the corect number of "full" evals.

i think the 1.1 comes from the fact that the default overhead (i.e fitness_computation_ratio) for fitness predictors is 0.1